src/platform/plugins/private/vis_types/vislib/public/vislib/components/legend/legend.tsx

```diff
@@ -104,8 +105,13 @@ export class VisLegend extends PureComponent<VisLegendProps, VisLegendState> {
       return false;
     }
 
-    const filters = await getDataActions().createFiltersFromValueClickAction({ data: item.values });
```

Comment:
rather than trying to create filters with a method that is not necesarry the one used in the actual uiAction to determine if uiAction is available, we should rathe use the hasCompatibleActions check